homeassistant/components/light/__init__.py

```diff
@@ -107,6 +110,7 @@
     ATTR_WHITE_VALUE: vol.All(vol.Coerce(int), vol.Range(min=0, max=255)),
     ATTR_FLASH: vol.In([FLASH_SHORT, FLASH_LONG]),
     ATTR_EFFECT: cv.string,
+    vol.Optional(ATTR_POWER_ON, default=True): cv.boolean,
```

Comment:
Yeah, I think I touched on both points in the "Alternatives" section. The idea comes up repeatedly because it's really useful and we miss it when converting from IFTTT 😄 

I'm cool with starting out with a LIFX only service (though that will not fix the Flux+motion sensor race). So that will indeed be something like `lifx_set_state` – or maybe `lifx_set_color`. I will look into that.

If I am creating a new service call, will it be acceptable to use the units that are well-known from the LIFX smartphone app? This is Kelvin for temperature and percentage for brightness.